src/component/base/__tests__/DraftEditor.react-test.js

```diff
@@ -50,3 +51,62 @@ test('must has editorKey same as props', () => {
     shallow._instance.getEditorKey || shallow._instance._instance.getEditorKey;
   expect(getEditorKey()).toMatchSnapshot();
 });
+
```

Comment:
We have moved away from describe in favor of root tests to avoid indentation. we can always make as a root test and add more information to the test title itself